Mathlib/GroupTheory/QuotientGroup.lean

```diff
@@ -631,4 +631,10 @@ noncomputable def fintypeOfKerOfCodom [Fintype g.ker] : Fintype G :=
 noncomputable def fintypeOfDomOfCoker [Normal f.range] [Fintype <| G ⧸ f.range] : Fintype G :=
   fintypeOfKerLeRange _ (mk' f.range) fun x => (eq_one_iff x).mp
 
+@[to_additive]
+lemma _root_.Finite.of_finite_quot_finite_subgroup {H : Subgroup G}
+    (hH : Set.Finite (H : Set G)) (h : Finite (G ⧸ H)) : Finite G := by
```

Comment:
Is there a reason to prefer `Set.Finite (H : Set G)` over `Finite H`?

Reply:
No reason, ends up being the same. I thought that since a subgroup is often treated more like a set like term rather than a type, that the Set level finite predicate made more sense. While for the ambient group and the quotient group, since they are treated more as types, the Finite predicate made more sense. Which would you prefer? I have no resistance to switching.

Reply:
I think `Finite` is probably preferable, even for subgroups, since it allows you to apply results from the group theory library directly to the subgroup without having to convert between the two notions of finite.

Reply:
I've changed it to take `[Finite H]`, which made the Ideal version require some more assistance. Please let me know if this works for you!

Reply:
Yes, great!